beamer/events.py

```diff
@@ -32,6 +32,11 @@ class LatestBlockUpdatedEvent(Event):
     block_data: BlockData
 
 
+@dataclass(frozen=True)
+class InitiateL1ResolutionEvent(Event):
```

Comment:
When is this event going to be created? Will this event unconditionally execute the L1 tx?